SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py

```diff
@@ -258,11 +221,6 @@
         AddInefficiency = False,
         AddThresholdSmearing = False,
     ),
-    Pixel3DDigitizerAlgorithm = dict(
```

Comment:
@SuchandraDutta,
This reverts https://github.com/cms-sw/cmssw/commit/f38453e10ec5e714f14c2fd0cb169eab4f32ec8a from @duartej. Please restore it as it was previously.